source/instance.txt

```diff
@@ -33,7 +33,7 @@ MongoDB instance to which Compass is connected, including:
    :alt: Compass Home View
 
 Once connected to a MongoDB instance, you can reach the Compass
-Home screen by clicking the :guilabel:`Home` icon in the upper left
+Home screen by clicking the cluster name in the upper left
```

Comment:
I'd suggest moving this descriptor above the image so users understand what we're highlighting.